INSTALL.md

```diff
@@ -63,38 +62,40 @@ checks locally before uploading your pull request.
 In order to install golangci-lint, you can run:
 
 ```
-$ curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh |
-  sh -s -- -b $(go env GOPATH)/bin v1.15.0
+# binary will be $(go env GOPATH)/bin/golangci-lint
+curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin vX.Y.Z
+
+# if GOPATH not set you can also install it into ./bin/ 
+curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s vX.Y.Z
+
 ```
 
 This will download and install golangci-lint from its Github releases
-page (using version v1.15.0 at the moment).
+page (using version v1.17.1 at the moment).
 
 ## Clone the repo
 
 Golang is a bit finicky about where things are placed. Here is the correct way
 to build Singularity from source:
 
 ```
-$ mkdir -p ${GOPATH}/src/github.com/sylabs && \
-  cd ${GOPATH}/src/github.com/sylabs && \
-  git clone https://github.com/sylabs/singularity.git && \
+$ git clone https://github.com/sylabs/singularity.git && \
   cd singularity
 ```
 
 To build a stable version of Singularity, check out a [release tag](https://github.com/sylabs/singularity/tags) before compiling:
 
 ```
-$ git checkout v3.3.0
+$ git checkout v3.4.0
 ```
 
 ## Compiling Singularity
 
 You can build Singularity using the following commands:
 
 ```
-$ cd ${GOPATH}/src/github.com/sylabs/singularity && \
-  ./mconfig && \
+# cd into your the source code folder and
```

Comment:
Does this make sense? We did `cd singularity` so we could `git checkout` above

If it's kept, it needs to be either `your` or `the`, not both.